l2geth/eth/downloader/downloader.go

```diff
@@ -1792,6 +1792,8 @@ func (d *Downloader) qosTuner() {
 		conf = conf + (1000000-conf)/2
 		atomic.StoreUint64(&d.rttConfidence, conf)
 
+		// Log the new QoS values and sleep until the next RTT
```

Comment:
I think that keeping the diff here is worth it, this logline makes the logs noisy